modules/nf-core/centrifuge/build/main.nf

```diff
@@ -15,21 +15,23 @@ process CENTRIFUGE_BUILD {
     path size_table
 
     output:
-    tuple val(meta), path("*.cf") , emit: cf
+    tuple val(meta), path("${prefix}/") , emit: cf
     path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
No def is needed here?

Reply:
No, otherwise can't access 'prefix' i nteh output scope above :)